ckanext/stats/blueprint.py

```diff
@@ -14,17 +14,23 @@
 @stats.route(u'/stats')
 def index():
     stats = stats_lib.Stats()
+    # Public stats
     extra_vars: dict[str, Any] = {
         'largest_groups': stats.largest_groups(),
         'top_tags': stats.top_tags(),
-        'top_package_creators': stats.top_package_creators(),
         'most_edited_packages': stats.most_edited_packages(),
         'new_packages_by_week': stats.get_by_week('new_packages'),
         'deleted_packages_by_week': stats.get_by_week('deleted_packages'),
         'num_packages_by_week': stats.get_num_packages_by_week(),
-        'package_revisions_by_week': stats.get_by_week('package_revisions')
+        'package_revisions_by_week': stats.get_by_week('package_revisions'),
+        # Stats with users info
+        'top_package_creators': None,
```

Comment:
To keep the style of the function let's keep calling `stats.top_package_creators()`  directly here instead of initializing it with `None`. 

All the logic for checking the config is encapsulated in the method so it is safe. We can avoid an extra `if` statement.

Reply:
You're right, done here https://github.com/avdata99/ckan/commit/b6324d96709192a4dbaf49e121cf7be9ad6384cd